expo-zustand-styled-components/src/components/Profile/UserCard/UserCard.styles.ts

```diff
@@ -6,25 +6,26 @@ type ScreenWidth = {
 };
 
 export const UserProfileCardWrapper = styled.View<ScreenWidth>`
-  flex-grow: 0;
-  display: flex;
-  padding-right: ${({ screenWidth }) => (screenWidth >= breakpoints.tablet ? 2.5 : 0)}%;
-  margin-top: ${({ screenWidth }) => (screenWidth >= breakpoints.tablet ? -20 : 0)}px;
+  z-index: unset;
   width: ${({ screenWidth }) => (screenWidth >= breakpoints.tablet ? 33.3 : 100)}%;
+  margin-top: ${({ screenWidth }) => (screenWidth >= breakpoints.tablet ? -80 : 0)}px;
+  padding-right: ${({ screenWidth }) => (screenWidth >= breakpoints.tablet ? 2.5 : 0)}%;
 `;
 
 export const UserDetails = styled.View<ScreenWidth>`
   gap: 8px;
+  z-index: unset;
```

Comment:
Expected style "zIndex: unset" to be unitless 
That's the error

Reply:
This is a warning from the style component that doesn't make any sense. as z-index can be a text, but then it is expecting it to be just numbered. This warning also appears in the web console. but it isn't an error.

Reply:
It's failing to load on my phone

Reply:
Did you test with your phone?

Reply:
And? WHat did you see?

Reply:
No issue on my end put i have been able to remove it for mobile